deploy-service/teletraanservice/src/main/java/com/pinterest/teletraan/resource/EnvStages.java

```diff
@@ -97,6 +101,19 @@ public EnvironBean get(
         return Utils.getEnvStage(environDAO, envName, stageName);
     }
 
+    @GET
+    @Path("/keelPipeline")
```

Comment:
Could it make sense to use `/pindeployPipeline` in the url to be consistent when the rest of the terminology used here?

Reply:
yes, updated, thank you